AlembicImporter/Packages/com.unity.formats.alembic/Runtime/Scripts/Exporter/AbcAPI.cs

```diff
@@ -390,10 +390,10 @@ internal static class NativeMethods
         [DllImport(Abci.Lib)] public static extern void aiObjectSetEnabled(IntPtr obj, Bool v);
         [DllImport(Abci.Lib)] public static extern IntPtr aiObjectGetName(IntPtr obj);
         [DllImport(Abci.Lib)] public static extern IntPtr aiObjectGetFullName(IntPtr obj);
-        [DllImport(Abci.Lib)] public static extern aiXform aiObjectAsXform(IntPtr obj);
-        [DllImport(Abci.Lib)] public static extern aiCamera aiObjectAsCamera(IntPtr obj);
-        [DllImport(Abci.Lib)] public static extern aiPoints aiObjectAsPoints(IntPtr obj);
-        [DllImport(Abci.Lib)] public static extern aiPolyMesh aiObjectAsPolyMesh(IntPtr obj);
+        [DllImport(Abci.Lib)] public static extern Sdk.aiXform aiObjectAsXform(IntPtr obj);
+        [DllImport(Abci.Lib)] public static extern Sdk.aiCamera aiObjectAsCamera(IntPtr obj);
```

Comment:
why does it need the Sdk here?

Reply:
Because I moved methods from another file to this file, and I defined these methods into structs that have the same name (e.g. NativeMethods.aiXform). We still want the methods you highlighted to resolve to the original structs (e.g. Sdk.aiXform).